docs/solidity-by-example.rst

```diff
@@ -645,4 +645,499 @@ Safe Remote Purchase
 Micropayment Channel
 ********************
 
-To be written.
+Creating and verifying signatures
+=================================
+
+Signatures are used to authorize transactions,
+and they're a general tool that's available to
```

Comment:
Could you please change the style to not use contractions (i.e. `they are a general tool that is available...') as in most of the rest of the documentation?